.clj-kondo/config.edn

```diff
@@ -686,9 +686,10 @@
   ;; to the new stricter rules from day 1
   metabase-lib
   {:linters
-   {:docstring-leading-trailing-whitespace {:level :warning}
-    :reduce-without-init                   {:level :warning}
-    :used-underscored-binding              {:level :warning}
-    :single-key-in                         {:level :warning}
-    :keyword-binding                       {:level :warning}
-    :shadowed-var                          {:level :warning}}}}}
+   {:docstring-leading-trailing-whitespace                 {:level :warning}
+    :reduce-without-init                                   {:level :warning}
+    :used-underscored-binding                              {:level :warning}
+    :single-key-in                                         {:level :warning}
+    :keyword-binding                                       {:level :warning}
+    :shadowed-var                                          {:level :warning}
+    :metabase/deftest-not-marked-parallel-or-synchronized  {:level :warning}}}}}
```

Comment:
I added a custom Kondo linter to require that `metabase.lib.*` tests are marked either `^:parallel` or `^:synchronous` if for some reason you don't want them to be parallel. Hopefully this will help us remember to make them all `^:parallel` and thus **fast**